zenodo/modules/spam/views.py

```diff
@@ -248,10 +251,13 @@ def safelist_admin():
             sa.func.max(Community.description).label('description')
         ).join(Community).group_by(User.id).filter(
             Community.created.between(from_date, to_date),
-            Community.deleted_at == None,
+            Community.deleted_at is None
```

Comment:
```suggestion
            Community.deleted_at.is_(None),
```

This is a weird behaviour from SQLAlchemy... The one you had resolves to `False`, which causes some weird behaviour in the produced SQL.